src/tree/GitHubConfigGroupTreeItem.ts

```diff
@@ -49,12 +53,18 @@ export class GitHubConfigGroupTreeItem extends AzExtParentTreeItem {
 
             for (const yamlFile of yamlFiles) {
                 const yamlFilePath: string = join(workflowsDir, yamlFile);
-                const buildConfigs: BuildConfigs | undefined = await parseYamlFile(context, yamlFilePath);
-                buildConfigs && treeItems.push(new GitHubConfigGroupTreeItem(parent, yamlFilePath, buildConfigs));
+                try {
+                    const buildConfigs: BuildConfigs | undefined = await parseYamlFile(context, yamlFilePath);
+                    buildConfigs && treeItems.push(new GitHubConfigGroupTreeItem(parent, yamlFilePath, buildConfigs));
+                } catch (e) {
+                    if (e instanceof YAMLSyntaxError) {
```

Comment:
We should still have tree items for when the error is not `YAMLSyntaxError`.  In those cases, you can just make `parseError(e).message` the label.